tests/Stripe/ObjectTest.php

```diff
@@ -1,12 +1,12 @@
 <?php
 
-class Stripe_ObjectTest extends UnitTestCase
+class Stripe_ObjectTest extends PHPUnit_Framework_TestCase
```

Comment:
`Stripe_TestCase`

I know the 1:1 translation of all of these would have them left as the base test case, but I see no reason not to inherit everything from `Stripe_TestCase` and give us the consistency there.